qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -13,6 +13,18 @@
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
     balloon_device = virtio-balloon-pci
+    Windows:
+        cdroms = "virtio"
+        # Please change following cdrom_virtio to the right version
+        # if necessary
+        cdrom_virtio = isos/windows/virtio-win.iso
+        # balloon service related cmd, which will be covered by different
+        # guest's paths in guest-os cfg.
+        install_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -i"
+        uninstall_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -u"
+        status_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe status"
+        run_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -r"
+        stop_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -s"
```

Comment:
If it's better to move those cfg to with_balloon_service as they are only available for balloon service ?

Reply:
I think it should be ok, any new variant can reuse these parameters in future, no need to rewrite again